website/client/components/appFooter.vue

```diff
@@ -1,46 +1,72 @@
-<template lang="pug">
+<template lang='pug'>
```

Comment:
I don't remember exactly why but initally we had problems with vue + pug when using single quotes for attributes